client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1605,6 +1605,10 @@ Undocumented.prototype.validateNewUser = function( data, fn ) {
  */
 Undocumented.prototype.requestMagicLoginEmail = function( data, fn ) {
 	restrictByOauthKeys( data );
+
+	data.locale = i18n.getLocaleSlug();
```

Comment:
I wondered what that returns for `en` locale, but @Tug said it should return `en` because of [this](https://github.com/Automattic/wp-calypso/blob/master/client/boot/common.js#L153).